docs/1_Getting-Started/1_getting-started.md

```diff
@@ -1,5 +1,9 @@
 # Getting Started
 
+## Docker
+
+Docker is the recommended way to use the LUCI ROS2 SDK. Instructions for how to get started with the docker container can be found in [Getting started with Docker](./2_docker.md)
+
 ## Setting up your development machine
 
 Due to the nature of the current packages, some require dependencies before they can be installed or run. These packages are built and tested to support ROS2 Galactic with Ubuntu 20.04. New packages can be developed to fit other ROS2 version and OS needs in the future.
```

Comment:
Please add "this can be skipped if using docker above"

Reply:
done